src/js/formapp.js

```diff
@@ -46,6 +46,9 @@ function renderForm({ formId, actionId, patientId }) {
         .then(form => {
           form.nosubmit = true;
           form.submission = { data: fields.data.attributes };
+          form.on('render', () => {
+            form.redraw();
+          });
```

Comment:
so I _think_ this is what's triggering the infinite loop. But it's only doing it in Firefox from what I can tell (and it's only a warning so...). Chrome is fine, and performance is fine as well, so it might be ok? Looking at the failed test now

Reply:
Yeah, sorry for my crappy code. Originally this was an on change, but this would redraw the form everytime anything was changed and causing issues with the user experience. This redraw is to convert code tags in the HTML blocks. What's a better way to ensure it only does that once after the form is loaded?

Reply:
I don't think an event is needed here.  I think it can just be:
```
form.submission = 
form.redraw();
```